channels/email/index.rst

```diff
@@ -67,6 +67,9 @@ Control how Zammad **sends and receives email**.
 
       .. warning:: 🤓 This is a very advanced topic.
 
+:doc:`🛟 Common M365 errors </channels/microsoft365/common-errors>`
+   Here you can find common errors and how to fix them.
+
```

Comment:
I think this block belongs to `channels/microsoft365/index.rst` instead. This way it might show up for other email channels where it's irrelevant. Just include it in that file *after* the `.. include` command that references this one.